src/Symfony/Component/Console/Application.php

```diff
@@ -860,6 +861,18 @@ protected function doRunCommand(Command $command, InputInterface $input, OutputI
                 $this->dispatcher->dispatch(ConsoleEvents::TERMINATE, $event);
 
                 throw $e;
+            } catch (\Throwable $e) {
+                $e = new FatalThrowableError($e);
+
+                $event = new ConsoleExceptionEvent($command, $input, $output, $e, $e->getCode());
+                $this->dispatcher->dispatch(ConsoleEvents::EXCEPTION, $event);
+
+                $e = $event->getException();
+
+                $event = new ConsoleTerminateEvent($command, $input, $output, $e->getCode());
+                $this->dispatcher->dispatch(ConsoleEvents::TERMINATE, $event);
+
+                throw $e;
```

Comment:
I think it would be better to just add another function and call that at the end of each try catch?